build.cake

```diff
@@ -364,19 +364,20 @@ Task("CreateLinuxPackages")
 
     DeleteDirectory(artifactsDir + $"/OctopusTools.{nugetVersion}.linux-x64.extracted", new DeleteDirectorySettings { Recursive = true, Force = true });
  
-    CreateDirectory($"{artifactsDir}/linuxpackages");
-    MoveFiles(GetFiles($"{artifactsDir}/*.deb"), $"{artifactsDir}/linuxpackages");
-    MoveFiles(GetFiles($"{artifactsDir}/*.rpm"), $"{artifactsDir}/linuxpackages");
-    CopyFileToDirectory($"{linuxPackageFeedsDir}/publish-apt.sh", $"{artifactsDir}/linuxpackages");
-    CopyFileToDirectory($"{linuxPackageFeedsDir}/publish-rpm.sh", $"{artifactsDir}/linuxpackages");
-    CopyFileToDirectory($"{assetDir}/repos/test-octopuscli-feed-packages.sh", $"{artifactsDir}/linuxpackages");
-    CopyFileToDirectory($"{assetDir}/repos/test-octopuscli-feed-package.sh", $"{artifactsDir}/linuxpackages");
-    CopyFileToDirectory($"{linuxPackageFeedsDir}/test-env-docker-images.conf", $"{artifactsDir}/linuxpackages");
-    CopyFileToDirectory($"{linuxPackageFeedsDir}/install-linux-feed-package.sh", $"{artifactsDir}/linuxpackages");
-    TarGzip($"{artifactsDir}/linuxpackages", $"{artifactsDir}/OctopusTools.Packages.linux-x64.{nugetVersion}");
+    var linuxPackagesDir = $"{artifactsDir}/linuxpackages";
```

Comment:
Suggest adding a breadcrumb to help future readers demystify where the value `"linux-package-feeds"` comes from. e.g. Add a comment here: https://github.com/OctopusDeploy/OctopusCLI/pull/83/files#diff-4d3634747586b964df9f7712bc3ec669R35, linking to the `linux-package-feeds` repo and the TeamCity artifact config (https://build.octopushq.com/admin/editDependencies.html?id=buildType:OctopusDeploy_OctopusCLI_BuildLinuxContainer)?

Reply:
Great idea!